sentry/src/test/java/io/sentry/TransactionContextsTest.kt

```diff
@@ -0,0 +1,53 @@
+package io.sentry
+
+import io.sentry.protocol.App
+import io.sentry.protocol.Browser
+import io.sentry.protocol.Device
+import io.sentry.protocol.Gpu
+import io.sentry.protocol.OperatingSystem
+import io.sentry.protocol.SentryRuntime
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNotSame
+
+class TransactionContextsTest {
+
+    @org.junit.Test
```

Comment:
can we use `import kotlin.test` package only? they are actually aliases to `@org.junit`